rest-api-spec/src/main/resources/rest-api-spec/api/security.query_api_key.json

```diff
@@ -0,0 +1,30 @@
+{
+  "security.query_api_key":{
```

Comment:
Since this API is very similar to `search` in that it has a top-level `query` object what are your thoughts on renaming to `search_api_key[s]`?

Reply:
We discussed about the name previously and `query` was picked over `search` on purpose. This is because regular search does a lot more than just query, e.g. aggregation, highlighting etc. The name query is indicative that this API only supports the `query` field of the regular search API. The existing [query watches API](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/watcher.query_watches.json) was also named "query" for the same reasons.

I am OK to use the plural form, i.e. `query_api_keys`, which is also more consistent with `query_watches`.

Reply:
Sounds good to me! Thanks for making me aware of `query_watches`, hadn't used that API before. Will keep this prefix in mind for the future.